blockdevice/util/disk/disks.go

```diff
@@ -88,6 +88,8 @@ type Disk struct {
 	UUID string
 	// Type is the disk type: HDD, SSD, SD card, NVMe drive.
 	Type Type
+	// FullPath is the full path of the symlinked device folder.
+	FullPath string
```

Comment:
It might be nice to call it `BusPath` as it's path to the device via device tree (and bus)